www/src/pages/ru/folder-structure.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Файловая структура
+description: Файловая структура нового T3 приложения
+layout: ../../layouts/docs.astro
+lang: ru
+---
+
+Так выглядит файловая структура нового T3 приложения, с выбранными всеми опциями.
+
+Ниже - описание каждой папки, указывающее на её назначение и когда она добавляется.
+
+```
```

Comment:
Hi, not a russian translator or maintainer. Just wanted to remind you to update the structure of this page to sync it with the english docs. It got updated :)

https://github.com/t3-oss/create-t3-app/commit/ec371627ea29d5fcc77e1be4228f85e2c557b742

Reply:
Yep on it

Reply:
Resolved in https://github.com/t3-oss/create-t3-app/pull/774/commits/2001b9ed6b430f8ed6f9ad805ce16b419c4552f3